README.md

```diff
@@ -1,6 +1,21 @@
 # MatchBox
 A simple application that uses the ArchivesSpace API to help determine the box into which a three-dimensional object will fit. A demonstration site is available at [https://github.com/RockefellerArchiveCenter/MatchBox](https://github.com/RockefellerArchiveCenter/MatchBox)
 
+## What is MatchBox?
+
+MatchBox is a browser-based tool that matches archival objects to appropriate box sizes. This collections management tool is used for housing materials that require box sizes more specialized than standard record storage boxes or standard letter/legal document boxes.
+
+Materials that often require specialized box sizes include realia and large or bulky documents such as flat files or scrapbooks. These archival objects are often considered to be oversize and/or three dimensional. MatchBox allows the individual working with such items to find boxes that will work best for long-term storage and preservation.  
+
+Additionally, MatchBox is used for calculating cubic footage of container profile(s). These calculations are necessary for space management and determining the extent of collections for inclusion in accession and resource records.
+
+## How it Works
+MatchBox displays a list of all container profiles recorded in ArchivesSpace along with their dimensions. This list is constantly growing as members of Collections Management add new container profiles to the database. MatchBox displays this data through the ArchivesSpace API, ensuring that the container profile information displayed is up to date.
+
+The dimensions of an object are entered into their respective fields: depth (length), width, and height. The results will populate, eliminating container profile options that are too small. If the results exclude adequate container options, there is typically no appropriate container profile recorded in ArchivesSpace. Contact a member of Collections Management to locate or order a specific box size. The dimensions of newly acquired container sizes will be added to the database for future searching.
```

Comment:
This looks great! I recommend removing "Contact a member of Collections Management to locate or order a specific box size" since this is for an external audience.